src/app/pages/dashboard/dashboard.component.html

```diff
@@ -27,3 +27,21 @@
     </nb-card>
   </div>
 </div>
+<div class="row">
```

Comment:
I don't think you need an extra row.
it should create a new row if the grid has no more space in the current row

Reply:
it does need to be in new section.
I want it to be divided